packages/nodes-base/nodes/Zammad/descriptions/UserDescription.ts

```diff
@@ -0,0 +1,658 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const userDescription: INodeProperties[] = [
+	// ----------------------------------
+	//           operations
+	// ----------------------------------
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a user',
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete a user',
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Retrieve a user',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Retrieve all users',
+			},
+			{
+				name: 'Get Self',
+				value: 'getSelf',
+				description: 'Retrieve currently logged-in user',
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update a user',
+			},
+		],
+		default: 'create',
+	},
+
+	// ----------------------------------
+	//             fields
+	// ----------------------------------
+	{
+		displayName: 'Email Address',
+		name: 'email',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'create',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'User Name/ID',
+		name: 'id',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'loadUsers',
+		},
+		description: 'User to update. Choose from the list or specify an ID using an <a href="https://docs.n8n.io/nodes/expressions.html#expressions">expression</a>.',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'user',
+				],
+				operation: [
+					'update',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'User Name/ID',
+		name: 'id',
+		type: 'options',
+		typeOptions: {
```

Comment:
- Because the entities exist in a fetchable number. Unlike an overly large number like Elasticsearch docs.
- Because the user can easily pick the entity, for better UX.
- Because the guide does not forbid this.